tests/test_obs_img_utils_cli.py

```diff
@@ -237,7 +237,7 @@ def test_image_download(
               'apparmor-parser\n'
               '\n'
               '2.12.2\n'
-              'lp150.6.14.1\n'
+              '\n'
```

Comment:
Here is an issue. I could only make this test to work if I removed the release information. Otherwise click.testing said the version would be invalid. On py3.6 it seems to work. Thus I expect some code issue here too.

@smarlowucf if you have a minute to double check on this test it would be great as I'm running out of time

Reply:
Ah, it seems the python version compare function no longer will accept this version style. For now I think we can drop the lp from test data. It exists in a number of spots, I had to modify the following files to get the warning to go away:

```
	modified:   tests/data/openSUSE-Leap-15.0-Azure.x86_64-1.0.0-Build1.133.packages
	modified:   tests/data/report
	modified:   tests/test_obs_img_utils_api.py
	modified:   tests/test_obs_img_utils_cli.py
	modified:   tests/test_obs_img_utils_utils.py
```

Reply:
I added a patch to move away from pkg_resources, this is deprecated. However all python version parsing we will use is always PEP440 compliant and SUSE package names are not. Thus I think this is an implementation mistake. We need to use a version parser that is compliant to RPM. But that's to be fixed in another PR. I will open an issue for this part